### PR TITLE
update UAL logo

### PR DESCRIPTION
**Pull request by pgwillia**
As of October 23, we are now the University of Alberta Library.

About this Dale Askey said, "I welcome having a name that accurately reflects our identity and our mission to serve the University of Alberta. The ability that all of you demonstrate to pull together to instigate, organize, and develop myriad cross-organizational activities is one of our great strengths, now reflected in our name."

image

#1616 

## What's New

Updated the ual-logo.png from the Media toolkit.

### CODE REVIEW

**Comment by mbarnett** [comment]
Changes to the various license/rights strings _probably_ (maybe?) need to come from legal? The DOI thing I wouldn't change unless it comes as a request from someone, because I'm not sure about what that may or may not need to match on Datacite's end or with other services...

**Comment by pgwillia** [comment]
Thanks @murny.  I made the changes noted.

File: config/locales/en.yml [hunk 1]
```diff
@@ -501,7 +501,7 @@ en:
       theses_collection: 'University of Alberta Theses and Dissertations'
       afns_collection: 'Department of Agriculture, Food and Nutritional Science Journal Articles'
       era_av_header: 'Explore ERA Audio+Video'
-      era_av_text: "The University of Alberta Libraries' Media Streaming Service"
+      era_av_text: "The University of Alberta Library' Media Streaming Service"
```
Comment (murny) [inline config/locales/en.yml:504]: Got an extra `'` here

Comment (pgwillia) [reply config/locales/en.yml]: Really glad to have your extra eyes.  I'll try to be more careful.

**Pull request merged**